docs/using-bi-for-auth.md

```diff
@@ -62,12 +62,29 @@ A full authorization request url has additional parameters that we need to accou
 https://auth-us.beyondidentity.com/v1/tenants/<tenant_id>/realms/<realm_id>/applications/<application_id>/authorize?
 response_type=code
 &client_id=<client_id_from_application>
-&redirect_uri=<redirect_uri from application>
+&redirect_uri=<redirect_uri_from_application>
 &scope=openid
 &state=<state>
+&code_challenge_method<method_from_PKCE_for_public_clients>
+&code_challenge<challenge_from_PKCE_for_public_clients>
```

Comment:
These are not formatting correctly because the '=' is missing 🙂

Reply:
thanks!